### PR TITLE
Completed javascript-conditionals exercise - Paxton Rosenberg

### PR DESCRIPTION
**Pull request by PaxtonRosenberg**

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    return 'acid';
+  } else if (pH >= 8 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+/* introduceWarnerBro Function */
+function introduceWarnerBro(name) {
```
Comment (robgmerrill) [inline javascript-conditionals/main.js:78]: The function is supposed to **return** a string. Same goes for the following function :)

**Review by robgmerrill — CHANGES_REQUESTED** [review]
A couple of functions are logging a string rather than returning a string as per the instructions :)

**Comment by PaxtonRosenberg** [comment]
image

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**